imports/plugins/core/ui-navbar/client/components/brand.js

```diff
@@ -23,10 +23,11 @@ class Brand extends Component {
       <a className="brand" onClick={this.handleClick}>
         {this.props.logo &&
           <div className="logo">
-            <img src={this.props.logo} />
+            {<img src={this.props.logo} />}
```

Comment:
Why the outer braces? The previous version is correct. I.e.
```js
<img src={this.props.logo} />

Reply:
Fixed